conf/storage-mongodb.properties

```diff
@@ -5,24 +5,24 @@
 #Variant
```

Comment:
Please do not include IP address or any other credentials/sensitive information in properties files.